README.md

```diff
@@ -108,8 +108,8 @@ Notable differences between ACCP and ACCP-FIPS:
 * ACCP uses [the latest release of AWS-LC](https://github.com/aws/aws-lc/releases), whereas, ACCP-FIPS uses [the fips-2022-11-02 branch of AWS-LC](https://github.com/aws/aws-lc/tree/fips-2022-11-02).
 * ACCP-FIPS builds AWS-LC in FIPS mode by passing `-DFIPS=1` when configuring AWS-LC's build.
 * In FIPS-mode, RSA keys are limited to 2048, 3072, or 4096 bits in size with public exponent F4.
-* ACCP-FIPS does not register SecureRandom by default due to the performance of AWS-LC’s entropy source in FIPS-mode. [A system property](https://github.com/corretto/amazon-corretto-crypto-provider#other-system-properties) is available to register SecureRandom from AWS-LC if needed, and the performance differences are described in further detail under the description of that property.
 * Due to the fact that an older branch of AWS-LC is used in FIPS-mode, there will be performance differences between ACCP and ACCP-FIPS. We highly recommend performing detailed performance testing of your application if you choose to experiment with ACCP-FIPS.
+* Before version 2.4.0, ACCP-FIPS did not register SecureRandom by default due to the performance of AWS-LC’s entropy source in FIPS-mode, with older versions of AWS-LC. Since version 2.4.0, ACCP-FIPS behaves as ACCP: it registers SecureRandom from AWS-LC by default.
```

Comment:
We should link to https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/FIPS.md, which provides details about the entropy sources used in the versions of AWS-LC.  I would highlight that ACCP uses AWS-LC-FIPS v2.0

Reply:
A few paragraphs above, it says "ACCP-FIPS is a variation of ACCP which uses AWS-LC-FIPS 2.x as its cryptographic module.".

Should we add the following sentence:

> For details about the FIPS module of AWS-LC-FIPS, including the entropy sources used, see the [AWS-LC FIPS.md documentation](https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/FIPS.md).

Reply:
^ I think that sentence makes sense to add.

Also, not sure how pedantic we need to be, but the "don't register SecureRandom in FIPS" behavior was [introduced in 2.1.0](https://github.com/corretto/amazon-corretto-crypto-provider/commit/dea1db225c18afc75f742a68611fb46f1e51a183), so we could be a little more precise here:

> Between versions 2.1.0 and 2.3.3 (inclusive), ACCP-FIPS did not register SecureRandom by default ...

Reply:
I've pushed a commit with those changes.